### ISSUE TITLE
Translate input in JoomlaUpdate field

### ISSUE THREAD

**Issue by joomlablr** [joomla/joomla-cms#46177]
### Steps to reproduce the issue
1. Go to /administrator/index.php?option=com_joomlaupdate&view=upload and you see blue button, called as "select file". This button has no translate in joomla language files. When your browser and whole system (win, macos, etc) in  for example - polish, but joomla language in English or other - button stay at polish.

### My proposition

in file administrator/components/com_joomlaupdate/tmpl/upload/default.php at rows 74 add some as 

<style>
    .custom-file-upload {
        display: inline-block;
        padding: 6px 12px;
        cursor: pointer;
        background-color: #007bff;
        color: #fff;
        border-radius: 4px;
    }

    #install_package {
        display: none;
    }
</style>
        <label for="install_package" class="custom-file-upload"> <?php echo Text::_('COM_INSTALLER_SELECT_FILE_BUTTON'); ?></label>
        
        <input class="form-control" type="file" id="install_package" name="install_package" accept=".zip,application/zip">

### Device and system info

* What version of Joomla were you using?
Joomla 5.3

* What device were you using? (eg, phone, laptop)
laptop

* What web browser were you using? (eg, Chrome, Safari)
Chrome, Mozilla

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by brianteeman** [comment]
As you have seen the text of the button for a field of type=file comes directly from your browser and it will be displayed in the language of your browser and not the language of the page you are viewing.

The only option to resolve this is to hide the native input and create a clone of the input which you can customise.

As we use a field of this type in many places your suggestion is probably not enough as we wouldnt want to copy paste the code everywhere it is used so the changes should be made further up the tree in either `libraries\src\Form\Field\FileField.php` or `layouts\joomla\form\field\file.php`
